app/src/processing/app/syntax/JEditTextArea.java

```diff
@@ -189,6 +190,7 @@ protected TextAreaPainter createPainter(final TextAreaDefaults defaults) {
   /**
    * Inline Input Method Support for Japanese.
    */
+  @Override
```

Comment:
Please don't add annotations like this, especially for classes that aren't affected by this PR.